configuration/pih/htmlforms/hiv/zl/hiv-dispensing.xml

```diff
@@ -337,23 +337,97 @@
-                       <tr>
-                           <td>
-                                <obs conceptId="PIH:1535"
-                                     answerConceptIds="PIH:3013,PIH:2848,PIH:3120"/>
-                           </td>
-                           <td>
-                               <!-- ToDo: revise this list with all the meds including tb and prophylaxis -->
-                                <obs conceptId="CIEL:1282" answerConceptSetIds="CIEL:1085" />
-                           </td>
-                           <td>
-                               <obs conceptId="CIEL:1443" />
-                           </td>
-                       </tr>
-                    </obsgroup>
-                </repeat>
+
+                <obsgroup groupingConceptId="CIEL:1442" >
+                <tr>
+                   <td>
+                       <uimessage code="ARV #1" />:
+                   </td>
+                   <td>
+                        <obs conceptId="CIEL:1282"
+                             answerDrugIds="
+                                78faac2e-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f968e6-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f9739a-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f95fa4-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f96a76-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                f2a2c2d9-16a6-4138-9074-6fdf3307e107,
+                                78f97cfa-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78fab02a-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f95bc6-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f967ba-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f960da-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78f96684-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                78faa9b8-dfbe-11e9-8a34-2a2ae2dbcce4,
+                                08d3817b-4e54-4825-bc4b-0512b627d567,
+                                0fc3d5c1-fd39-4899-b5c0-b094e28ff359,
+                                a6985b15-5fd6-4bdf-93f6-62930a438464"
+                             answerLabels="
+                                DTG/3TC/TDF 50/300/300 mg,
+                                AZT/3TC 300/150 mg,
+                                AZT/3TC/NVP 300/150/200 mg,
+                                AZT/3TC 60/30 mg,
+                                TDF/3TC 300/300 mg,
+                                ABC/3TC 300/300 mg,
+                                AZT/3TC/NVP 60/30/50 mg,
+                                DTG 50 mg,
+                                AZT 10 mg sp,
+                                NVP 10 mg sp,
+                                TDF/3TC/EFV 300/300/600 mg,
+                                3TC 10 mg sp,
+                                ABC/3TC 60/30 mg,
+                                RAL/DAR/r 400/600/100mg,
```

Comment:
Use DRV instead of DAR.  
![image](https://user-images.githubusercontent.com/1164849/93773992-ba9f1f00-fbee-11ea-8da5-34fe91b19c96.png)